apps/ws/src/GameManager.ts

```diff
@@ -33,7 +33,7 @@ export class GameManager {
   }
 
   removeUser(socket: WebSocket) {
-    const user = this.users.find((user) => user.socket !== socket);
+    const user = this.users.find((user) => user.socket === socket);
```

Comment:
@hkirat this wasn;t removing the user properly.

Reply:
does this really remove the user tho?
The real filter is below

Reply:
We are passing the user to SocketManager. Line 42.
This was causing the same user to be added multiple times on refreshing, which was crashing the game